src/app/lists/CommentBox.tsx

```diff
@@ -88,16 +88,16 @@ const CommentContainer: React.FC<CommentProps> = ({ originalComment }) => {
             setSuccessMessage("");
         } else {
             setErrorMessage("");
-            setResetComment(value);
             setSuccessMessage("Comment successfully updated.");
+            window.location.reload();
```

Comment:
just wondering if there's a way to do this with state (and forcing rerenders) instead of refreshes... it's fine to do it this way but if we can get rid of a whole page refresh it would be lovely!

Reply:
If I have time I will have a look!